SparkPost/2web2ui
# AC-921: Allow manually billed customers to switch to self serve billing

AC-921: Allow manually billed customers to switch to self serve billing

### What Changed
 - Billing page should display an "add a credit card" button for manually billed customers. (an action in the Banner component)
- Shows the UpdatePayment information form 
- After updating the information, customer should be self serve and brought to /account/billing

### How To Test
 - Create a manually billed account using https://github.com/SparkPost/sparkpost-admin-api-documentation/blob/master/services/billing_api.md#create-a-manually-billed-account-post .
- Login as self-serve user too and try migrating to /account/billing/enable-automatic, you should be redirected to 404.
- Check if login for different roles work.

### To Do
